README.md

```diff
@@ -166,6 +166,12 @@ This process updates PO files as necessary and compiles a list of strings that a
 **Remark:** The file `common/translation_utils/translations.csv` is NOT updated in the process.
 Even though it has a column "is in use", its data is currently not updated automatically, and similarly, new strings are not added automatically.
 
+
+## Updating show in menu for existing pages
```

Comment:
I want to keep the main readme fairly short and clear of information that won't be relevant to most people. Suggest we move this to a page in docs directory - perhaps a troubleshooting page - with a link to it from the main readme.

Reply:
It's moved to a page in the docs directory with a link from the main readme.